sw/device/tests/otbn_ecdsa_p256_test.c

```diff
@@ -151,16 +152,17 @@ static void setup_data_pointers(otbn_t *otbn_ctx) {
  * Signs a message with ECDSA using the P-256 curve.
  *
  * @param otbn_ctx            The OTBN context object.
+ * @param testutils_ctx       The OTBN test utils context object.
  * @param msg                 The message to sign (32B).
  * @param private_key_d       The private key (32B).
  * @param[out] signature_r    Signature component r (the x-coordinate of R).
  *                            Provide a pre-allocated 32B buffer.
  * @param[out] signature_s    Signature component s (the proof).
  *                            Provide a pre-allocated 32B buffer.
  */
-static void p256_ecdsa_sign(otbn_t *otbn_ctx, const uint8_t *msg,
-                            const uint8_t *private_key_d, uint8_t *signature_r,
-                            uint8_t *signature_s) {
+static void p256_ecdsa_sign(otbn_t *otbn_ctx, otbn_testutils_t *testutils_ctx,
```

Comment:
nit: perhaps `testutils_ctx` could be more specific to match `otbn_ctx`, e.g., `otbn_testutils_ctx`?